### ISSUE TITLE
Support for three lines within the List component

### ISSUE THREAD

**Issue by MintyMods** [hperrin/svelte-material-ui#47]
Hi @hperrin, 

Currently there is support for up to two lines within the [list component](https://sveltematerialui.com/demo/list). I have a requirement for three lines and can see that the underlying [MDC component](https://material.io/components/lists/#types) supports 3 lines so do you have plans on exposing this additional functionality? 

I think it would make a good 'first-issue' / 'help-wanted' if this is something you don't class as high priority.

Sorry to keep raising issues but that's what happens when people use all your cool shit 🥇

**Comment by MintyMods** [comment]
Hi Hunter,

I had a little play with this and I guess the reason you don't currently have the 3 line support is because @material does not currently support it. I found the following [issue asking for support](https://github.com/material-components/material-components-web/issues/31) which I have asked for an update on but it's like 3 years old. 

So I have added initial support via a small change to `List.svelte` to toggle the `mdc-list--three-line` class and adding the _'not yet implemented'_ styles from @material to _style.scss. This now allows me to use the three line variant by passing the `threeLine` attribute to give:-

![image](https://user-images.githubusercontent.com/2349765/66998849-dcce1a00-f0cc-11e9-9682-106b1fca40c4.png)

and the dense variant:-

![image](https://user-images.githubusercontent.com/2349765/66999355-c3799d80-f0cd-11e9-8ef3-b9a4a5e25d17.png)

If I can work out how to do a pull request I will submit my suggested fix for you to review.
